session.go

```diff
@@ -731,6 +777,18 @@ func (s *session) handleUnpackedPacket(packet *unpackedPacket, rcvTime time.Time
 		})
 	}
 
+	if containsIgnoredFrames && !containsValidFrames {
```

Comment:
I think it's probably simpler and more correct to reject the whole packet if any frames were rejected.  How hard would it be to ensure that within a packet either all frames are accepted or none are?